ChangeLog

```diff
@@ -34,6 +34,8 @@ Features
      The following functions from the ECDH module can be replaced
      with an alternative implementation:
      mbedtls_ecdh_gen_public() and mbedtls_ecdh_compute_shared().
+   * Add support for alternative implementation for ECJPAKE, controlled by
```

Comment:
```diff
- Add support for alternative implementation for ECJPAKE
+ Add support for alternative implementation of ECJPAKE
```

Reply:
@Patater That's my feeling as well, but I decided to follow the wording of the pre-existing entry on `ECDH` the precedes this new one.

Reply:
Don't copy wrong things...